src/tabs/logging.html

```diff
@@ -12,35 +12,35 @@
                 <dt>
                     <label><input type="checkbox" name="MSP_RAW_IMU" /> MSP_RAW_IMU</label>
                 </dt>
-                <dd>9 columns (accel[x, y, z], gyro[x, y, z], mag[x, y, z])</dd>
+                <dd>9 columns (gyro[x, y, z], accel[x, y, z], mag[x, y, z])</dd>
                 <dt>
                     <label><input type="checkbox" name="MSP_ATTITUDE" /> MSP_ATTITUDE</label>
                 </dt>
-                <dd>3 columns (x, y, z)</dd>
+                <dd>3 columns (kine[x, y, z])</dd>
                 <dt>
                     <label><input type="checkbox" name="MSP_ALTITUDE" /> MSP_ALTITUDE</label>
                 </dt>
-                <dd>one column</dd>
+                <dd>1 column (alt)</dd>
                 <dt>
                     <label><input type="checkbox" name="MSP_RAW_GPS" /> MSP_RAW_GPS</label>
                 </dt>
-                <dd>7 columns</dd>
+                <dd>7 columns (Fix, NumSat, Lat, Lon, Alt, Speed, GroundCourse)</dd>
                 <dt>
                     <label><input type="checkbox" name="MSP_ANALOG" /> MSP_ANALOG</label>
                 </dt>
-                <dd>4 columns</dd>
+                <dd>4 columns (voltage, amperage, mAhdrawn, rssi)</dd>
                 <dt>
                     <label><input type="checkbox" name="MSP_RC" /> MSP_RC</label>
                 </dt>
-                <dd>8 columns by default</dd>
+                <dd>8 columns (RC0, RC1, RC2, RC3, RC4, RC5, RC6, RC7)</dd>
                 <dt>
                     <label><input type="checkbox" name="MSP_MOTOR" /> MSP_MOTOR</label>
                 </dt>
-                <dd>8 columns by default</dd>
+                <dd>8 columns (Mot0,  Mot1, Mot2, Mot3, Mot4, Mot5, Mot6, Mot7)</dd>
```

Comment:
```suggestion
                <dd>8 columns (Mot, Mot2, Mot3, Mot4, Mot5, Mot6, Mot7, Mot8)</dd>
```

Reply:
Good idea, since this are my first PR, I hope I did it right, I presse resolved to agree the change.

Reply:
Sorry. my wrong it should be `Mot1`

Also please try squashing commits using https://github.com/betaflight/betaflight/wiki/Betaflight-Contribution
Amend also works great with force push for minor updates.

Reply:
Right Mo1 gives the right sequence